backend/apps/matching/models/participant_info_location.py

```diff
@@ -31,6 +32,8 @@ class AbstractParticipantInfoLocation(models.Model):
     Only for Germany and Austria.
     """
 
+    MAX_RADIUS = 100  # km - limits the search APIs radius
```

Comment:
That's 2.2 million people in Meck-Pom and 128.9 million people in Mega-Berlin.

Reply:
thats 5 people in Schleswig-Holstein

Reply:
but i would be ok if we make it a config ;)